.mage/docs.go

```diff
@@ -115,34 +108,10 @@ func (d Docs) Build() (err error) {
 	if err != nil {
 		return err
 	}
-	url.Path = path.Join(url.Path, currentVersion)
-	destination := path.Join("public", currentVersion)
-	defer func() {
-		genErr := d.generateRedirect()
-		if err == nil {
-			err = genErr
-		}
-	}()
+	destination := "public"
 	return execHugo("-b", url.String(), "-d", destination, "--environment", "gh-pages")
```

Comment:
This is the only place where `destination` is used, so no need to have it as a variable anymore.